examples/feedback/feedback-metrics-server/config/multiclass-deployment.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: seldon-multiclass-model-metrics
+  labels:
+    app: seldon-multiclass-model-metrics
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: seldon-multiclass-model-metrics
+  template:
+    metadata:
+      annotations:
+        prometheus.io/path: /v1/metrics
+        prometheus.io/scrape: "true"
+      labels:
+        app: seldon-multiclass-model-metrics
+    spec:
+      securityContext:
+          runAsUser: 8888
+      containers:
+      - name: user-container
+        image: seldonio/alibi-detect-server:1.4.0-dev
```

Comment:
same - 1.5.0-dev
Might want to work out a way to have these take of the currebt version.txt?